metadata-service/auth-impl/src/main/java/com/datahub/authentication/post/PostService.java

```diff
@@ -87,4 +92,35 @@ public boolean deletePost(@Nonnull Urn postUrn, @Nonnull Authentication authenti
     _entityClient.deleteEntity(postUrn, authentication);
     return true;
   }
+
+  public boolean updatePost(
+      @Nonnull Urn postUrn,
+      @Nonnull String postType,
+      @Nonnull PostContent updatedContent,
+      @Nonnull Authentication authentication)
+      throws RemoteInvocationException, URISyntaxException {
+
+    val response =
```

Comment:
Please use `final EntityResponse entityResponse` instead. We use specific types in the java code base

Reply:
sure, it is just to make it less verbose. But I will add explicit type here.